tests/spec/w3c/headers-spec.js

```diff
@@ -67,6 +67,16 @@ describe("W3C — Headers", () => {
         "has received the endorsement of the W3C and its Members. W3C recommends the wide deployment of this specification as a standard for the Web."
       );
     });
+
+    describe("specStatus - base", () => {
```

Comment:
The previous describe block is `specStatus`. Should we merge this test in that block?

Reply:
I thought of splitting them by type, as "base" is kinda special.